hawk/src/main/java/com/orhanobut/hawk/Hawk.java

```diff
@@ -27,8 +27,12 @@ public static HawkBuilder init(Context context) {
     if (context == null) {
       throw new NullPointerException("Context should not be null");
     }
-    hawkBuilder = new HawkBuilder(context);
-    return hawkBuilder;
+
+    return new HawkBuilder(context);
+  }
+
+  static void onHawkBuilt(HawkBuilder builder) {
```

Comment:
I dont like this approach. Better ideas? I need to notify Hawk that the HawkBuilder has been initialised and need to pass the instance so the HawkInternal can be created.